conf/aws.conf

```diff
@@ -0,0 +1,381 @@
+#
+# Simple AWS Cloudera Director configuration file with automatic role assignments
+# that works as expected if you use a single instance type for all cluster nodes
+#
+
+#
+# Cluster name
+#
+
+name: C5-Simple-AWS
+
+#
+# Cloud provider configuration (credentials, region or zone and optional default image)
+#
+
+provider {
+    type: aws
+
+    #
+    # Get AWS credentials from the OS environment
+    # See http://docs.aws.amazon.com/general/latest/gr/aws-security-credentials.html
+    #
+    # If specifying the access keys directly and not through variables, make sure to enclose
+    # them in double quotes.
+
+    accessKeyId: "%(accessKeyId)s"
+    secretAccessKey: "%(secretAccessKey)s"
+
+    #
+    # Whether to publish access keys. Default is false.
+    # Publishing access keys to clients is necessary for those clients to have access to S3.
+    # These keys are published in the configuration files of all HDFS clients.
+    #
+
+    publishAccessKeys: true
+
+    #
+    # ID of the Amazon AWS region to use
+    # See: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html
+    #
+
+    region: %(region)s
+
+    #
+    # Region endpoint (if you are using one of the Gov. regions)
+    #
+
+    # regionEndpoint: ec2.us-gov-west-1.amazonaws.com
+
+    #
+    # Name of the public key registered in AWS
+    # See: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html
+    #
+
+    keyName: %(keyName)s
+
+    #
+    # ID of the VPC subnet
+    # See: http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_Subnets.html
+    #
+
+    subnetId: %(subnetId)s
+
+    #
+    # Comma separated list of security group IDs
+    # See: http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_SecurityGroups.html
+    #
+
+    securityGroupsIds: %(securityGroupsIds)s
+
+    #
+    # A prefix that Cloudera Director should use when naming the instances (this is not part of the hostname)
+    #
+
+    instanceNamePrefix: cloudera-director
```

Comment:
I was trying to leave this eggo-agnostic, so we can move it out or use it separately.